src/Microsoft.DotNet.Build.Tasks/AddDependenciesToProjectJson.cs

```diff
@@ -198,15 +195,34 @@ private void AddPackageItemsToDictionary(ref Dictionary<string, PackageItem> pac
             }
         }
 
-        // A versions file is of the form https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/master/Latest_Packages.txt
+        private static async System.Threading.Tasks.Task<string> GetFileAsync(string uri)
+        {
+            string contents = string.Empty;
+            using (var stream = (File.Exists(uri)) ? File.OpenRead(uri) : await new HttpClient().GetStreamAsync(uri))
+            using (StreamReader reader = new StreamReader(stream))
+            {
+                contents = await reader.ReadToEndAsync();
+            }
+            return contents;
+        }
+
+        // A versions file is of the form https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/corefx/master/Latest_Packages.txt
```

Comment:
Maybe this is a question for a corresponding corefx PR, but should we pin this to a commit SHA for repeatable builds?